sharding-orchestration/sharding-orchestration-reg/sharding-orchestration-reg-nacos/src/main/java/org/apache/shardingsphere/orchestration/reg/nacos/NacosRegistryCenter.java

```diff
@@ -137,7 +138,7 @@ public void initLock(final String key) {
 
     @Override
     public boolean tryLock() {
-        return false;
+        return true;

```

Comment:
tryLock(), initLock(), close() these three method need to be implemented. And Unit tests for nacos registry need to be added to cover most methods.

Reply:
I know this will be used in the Leaf×××××××Generator，but Nacos has no distributed lock，so what should we do? Use ReentrantLock directly？